deploy/olm-catalog/mobile-developer-console-operator/0.2.6/mobile-developer-console-operator.v0.2.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,269 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments:
+      - name: mobile-developer-console-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: mobile-developer-console-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: mobile-developer-console-operator
+            spec:
+              containers:
+              - command:
+                - mobile-developer-console-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: mobile-developer-console-operator
+                - name: OPENSHIFT_HOST
+                  value: ${OPENSHIFT_HOST}
+                - name: UPS_DOCUMENTATION_URL
+                  value: https://docs.aerogear.org/limited-availability/downstream/ups.html
+                - name: IDM_DOCUMENTATION_URL
+                  value: https://docs.aerogear.org/limited-availability/downstream/idm.html
+                - name: SYNC_DOCUMENTATION_URL
+                  value: https://docs.aerogear.org/limited-availability/downstream/sync.html
+                - name: MSS_DOCUMENTATION_URL
+                  value: https://docs.aerogear.org/limited-availability/downstream/mss.html
+                image: quay.io/aerogear/mobile-developer-console-operator:0.3.0
```

Comment:
The version here is 0.3.0 The folder structure is using 0.2.6

Reply:
Currently the MDC manifest file is on 0.2.6, bumping the version would require generate another CSV file

Reply:
Good catch! I was initially deliberately not going to generate the 0.3.0 one so that we could just get this PR correct and then it would be a simple generation for 0.3.0 in a different PR. However, once I rebased onto the latest master, this image kept getting put in here like this (I manually modified it back, but I must have regenerated again to get it back in).

Given that this is the only new comment on this PR, I'll both fix this and generate the 0.3.0 in this PR now.